spec/support/grpc_helpers.rb

```diff
@@ -0,0 +1,64 @@
+require 'grpc'
+
+def run_service
```

Comment:
Not a major thing, but would it be more appropriate to put this method into a module? The module could then be composed into the RSpec examples using `include GRPCHelpers`, while having the benefit of not having a global level `run_service` method.